gnucash/gnome/gnc-budget-view.c

```diff
@@ -716,11 +716,25 @@ gnc_budget_view_save_account_filter (GncBudgetView *budget_view)
 
     priv = GNC_BUDGET_VIEW_GET_PRIVATE(budget_view);
 
-    // Save account filter state information to budget section
-    gnc_tree_view_account_save_filter (GNC_TREE_VIEW_ACCOUNT(priv->tree_view),
-                                       priv->fd, gnc_state_get_current (),
-                                       gnc_tree_view_get_state_section (
-                                       GNC_TREE_VIEW(priv->tree_view)));
+    if (GNC_IS_BUDGET(priv->budget))
+    {
+        const gchar *name = gnc_budget_get_name (priv->budget);
+        gchar *date = gnc_print_time64 (gnc_time (NULL), qof_date_format_get_string (QOF_DATE_FORMAT_ISO));
+        GKeyFile *key_file = gnc_state_get_current();
+        gchar *group_comment = g_strdup_printf ("Budget state for \"%s\" - %s", name, date);
```

Comment:
I was thinking it would be nice to know when it was last opened but in light of other comments it would not make sense for doing so just for budgets

Reply:
Well, to your credit you consistently added it for invoices and bills as well :-)

I'm just trying to see the use case for a date stored in a state file. This is something users normally never open. If they have to our design has failed. But I may be missing something of course, so I'm interested to hear what use you see for it?

Reply:
I was thinking that there might be a lot of state information saved for registers / budgets that have not been accessed in a while and hence one might add a routine to remove these groups based on date but with a move to using register types this would not be the case.